pkg/cleantemp/cleantemp.go

```diff
@@ -44,38 +51,35 @@ func CleanTempDir(ctx logContext.Context, dirName string, pid int) error {
 	}
 
 	for _, proc := range procs {
-		if strings.Contains(proc.Executable(), dirName) {
+		if proc.Executable() == execName {
```

Comment:
This is stricter check - have you tested it's still working as intended?

Reply:
I did not but it says ```	// Executable name running this process. This is not a path to the
	// executable.```

Reply:
I added a rough test to verify that.